openfga/v1/openfga_service.proto

```diff
@@ -968,7 +957,6 @@ message ReadAuthorizationModelsResponse {
   ];
 }
 
-// ReadTuples should only be used for the playground. do not enable it for prod deployments
 message ReadTuplesRequest {
```

Comment:
Can we mark the `ReadTuplesRequest` and `ReadTuplesResponse` messages as deprecated as well? It's probably not critical because they are bound to the `rpc ReadTuples` which is deprecated, but this way the generated code has deprecation notices for those generated structs as well.

Reply:
Done. Thanks!